### PR TITLE
Select2 Autocomplete support - Doc updated

### PR DESCRIPTION
**Pull request by aybarscengaver**

### CODE REVIEW

**Comment by aybarscengaver** [comment]
are you approve it ?

File: Resources/doc/index.md [hunk 1]
```diff
@@ -125,7 +157,9 @@ class DefaultController extends Controller
 The first action, ``searchBookAction``, is needed to search books and to display them
 inside your field. Here, a possible ``findLikeName`` repository method is used, to
 search with ``LIKE`` statement (e.g. "pe" will find "War and Peace").
-A possible twig template for first action:
+A possible twig template for first action :
```
Comment (garak) [inline Resources/doc/index.md]: Why this space?

Comment (aybarscengaver) [inline Resources/doc/index.md]: no reason, is it a problem?

Comment (garak) [inline Resources/doc/index.md]: Can you remove it?

Comment (aybarscengaver) [inline Resources/doc/index.md]: I can't understand where is problem/space, which line range ?

Comment (garak) [inline Resources/doc/index.md]: It's in the diff above, can't you see it? Line 128/160

Comment (aybarscengaver) [inline Resources/doc/index.md]: it isn't space, please look difference on file.

**Comment by garak** [comment]
http://pasteboard.co/2zZ3UM7H.png

**Comment by aybarscengaver** [comment]
http://pasteboard.co/2zZdYglc.png

**Comment by garak** [comment]
So, you introduced a space after "action". Can you remove it?

**Comment by aybarscengaver** [comment]
yes, if i understand where is space. 128-160 line range is not null. wait. did you mean 1 char space between action and colon ? oh my gosh. yes i do it.

**Pull request closed without merging**